doc/README.md

```diff
@@ -0,0 +1,103 @@
+The Coq documentation
+=====================
+
+The Coq documentation includes
+
+- A Reference Manual
+- A document presenting the Coq standard library
+
+The documentation of the latest released version is available on the Coq
+web site at [coq.inria.fr/documentation](http://coq.inria.fr/documentation).
+
+Additionnally, you can view the documentation for the current master version at
+<https://gitlab.com/coq/coq/-/jobs/artifacts/master/file/_install_ci/share/doc/coq/sphinx/html/index.html?job=documentation>.
+
+The reference manual is written is reStructuredText and compiled
+using Sphinx (see [`sphinx/README.rst`](/doc/sphinx/README.rst))
+to learn more about the format that is used.
+
+The documentation for the standard library is generated from
+the `.v` source files using coqdoc.
+
+Prerequisite
+------------
+
+To produce the complete documentation in HTML, the following tools are required:
+
+  - Python 3
+  - Sphinx 1.6.5 (http://www.sphinx-doc.org/en/stable/)
+  - sphinx_rtd_theme
+  - pexpect
+  - beautifulsoup4
+  - Antlr4 runtime for Python 3
+
+To produce the documentation in PDF and PostScript formats the following
+additional tools are required:
```

Comment:
which are installed as binaries (not Python packages).  (The reader should search for and install each tool)

Reply:
It would be nice to get these to work on Windows but probably not necessary for what I want to do.

Cygwin has a long list of packages that can be installed: (https://cygwin.com/packages/package_list.html) but it appears not to include the 4 listed here.  (cygwin packages are new to me)